build.gradle.kts

```diff
@@ -0,0 +1,68 @@
+plugins {
+    `kotlin-dsl`
+    kotlin("jvm") version embeddedKotlinVersion
+    `java-gradle-plugin`
+
```

Comment:
id "com.github.ben-manes.versions" version "0.43.0"
 id 'com.adarshr.test-logger' version '3.2.0'

Reply:
oh good catch, I forgot to re-add them!